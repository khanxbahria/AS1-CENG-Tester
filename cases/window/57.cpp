#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Window* wind = new Window();
wind->closeTab();
Tab G = Tab("f", "bb","iii");
wind->isEmpty();
Tab B = Tab("fff", "r","g");
wind->getActiveTab();
Tab Q = Tab("ii", "ff","t");
wind->getActiveTab();
Tab L = Tab("w", "xxx","bbb");
wind->getActiveTab();
Tab H = Tab("rr", "h","fff");
wind->print();
Tab O = Tab("aaa", "mmm","ttt");
wind->getActiveTab();
Tab W = Tab("aa", "e","u");
wind->moveActiveTabTo(0);
Tab T = Tab("ee", "xx","ccc");
wind->newTab(B);
Tab A = Tab("w", "b","o");
wind->print();
wind->newTab(A);
wind->changeActiveTabTo(3);
Tab E = Tab("zzz", "cc","mm");
wind->print();
Tab J = Tab("ee", "dd","fff");
wind->isEmpty();
Tab P = Tab("bb", "mmm","ttt");
wind->isEmpty();
wind->newTab(E);
wind->closeTab();
Tab Z = Tab("ooo", "qq","z");
wind->moveActiveTabTo(3);
Tab D = Tab("kkk", "s","yyy");
wind->print();
Tab I = Tab("yyy", "kkk","yyy");
wind->closeTab();
Tab N = Tab("yy", "q","t");
wind->moveActiveTabTo(5);
wind->changeActiveTabTo(3);
wind->changeActiveTabTo(2);
Tab U = Tab("lll", "mm","uu");
wind->getActiveTab();
wind->closeTab();
wind->closeTab();
Tab M = Tab("e", "eee","uuu");
wind->closeTab();
wind->closeTab();
Tab S = Tab("oo", "ppp","mm");
wind->closeTab();
wind->isEmpty();
wind->isEmpty();
wind->newTab(D);
wind->closeTab();
Tab Y = Tab("o", "ggg","tt");
wind->moveActiveTabTo(3);
Tab C = Tab("uuu", "w","vv");
wind->print();
wind->closeTab();
wind->newTab(C);
Tab X = Tab("ooo", "c","f");
wind->newTab(G);
wind->closeTab();
wind->print();
wind->moveActiveTabTo(4);
wind->newTab(H);
wind->getActiveTab();
wind->moveActiveTabTo(3);
wind->isEmpty();
wind->print();
Tab K = Tab("ttt", "xx","j");
wind->getActiveTab();
wind->moveActiveTabTo(5);
wind->isEmpty();
wind->closeTab();
Tab V = Tab("qqq", "yy","ggg");
wind->changeActiveTabTo(0);
wind->isEmpty();
wind->isEmpty();
wind->moveActiveTabTo(0);
wind->changeActiveTabTo(1);
wind->closeTab();
wind->getActiveTab();
wind->print();
wind->print();
Tab R = Tab("ggg", "p","ccc");
wind->changeActiveTabTo(2);
wind->getActiveTab();
wind->moveActiveTabTo(2);
wind->print();
wind->newTab(I);
wind->getActiveTab();
wind->print();
wind->getActiveTab();
wind->moveActiveTabTo(5);
wind->changeActiveTabTo(1);
wind->moveActiveTabTo(2);
wind->isEmpty();
wind->moveActiveTabTo(5);
wind->changeActiveTabTo(0);
wind->closeTab();
wind->moveActiveTabTo(4);
wind->newTab(J);
wind->print();
wind->changeActiveTabTo(5);
wind->newTab(K);
wind->changeActiveTabTo(5);
wind->moveActiveTabTo(4);
wind->isEmpty();
wind->changeActiveTabTo(4);
wind->getActiveTab();
wind->changeActiveTabTo(3);
wind->print();
wind->getActiveTab();
wind->closeTab();
wind->newTab(L);
wind->getActiveTab();
wind->newTab(M);
wind->print();
wind->getActiveTab();
wind->newTab(N);
wind->newTab(O);
wind->closeTab();
wind->moveActiveTabTo(2);
wind->isEmpty();
wind->print();
wind->getActiveTab();
Tab F = Tab("rrr", "nnn","vvv");
delete wind;
}
	return 0;
}