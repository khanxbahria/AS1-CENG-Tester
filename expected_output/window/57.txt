Window is empty
The list is empty.
The active tab is tab 0
r
fff
g

The active tab is tab 1
r
fff
g

b
w
o

The active tab is tab 1
r
fff
g

b
w
o

Window is empty
The list is empty.
The active tab is tab 0
w
uuu
vv

The active tab is tab 1
w
uuu
vv

h
rr
fff

Window is empty
The list is empty.
Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 0
kkk
yyy
yyy

The active tab is tab 0
dd
ee
fff

The active tab is tab 1
dd
ee
fff

xx
ttt
j

The active tab is tab 2
dd
ee
fff

xxx
w
bbb

eee
e
uuu

The active tab is tab 2
dd
ee
fff

xxx
w
bbb

q
yy
t

eee
e
uuu

